documentation/configuration.md

```diff
@@ -2053,3 +2134,45 @@ The following example includes a default egress provider that corresponds to the
     value: "monitorBlob"
   ```
 </details>
+
+## **[Experimental]** In-Process Features Configuration (7.0+)
+
+Some features of `dotnet monitor` require loading libraries that are not part of the deployment of the target application. These libraries ship with `dotnet monitor` and are provisioned to be available to the target applications using the `DefaultSharedPath` option in the [storage configuration](#storage-configuration) section. The following features require these in-process libraries to be used:
+
+- Call stack collection
+
+ Because these libraries are loaded into the target application, they may have performance impact on memory and CPU utilization. These features are off by default and may be enabled via the `InProcessFeatures` configuration section.
```

Comment:
```suggestion
Because these libraries are loaded into the target application, they may have performance impact on memory and CPU utilization. These features are off by default and may be enabled via the `InProcessFeatures` configuration section.
```